diff -aurN libcm-0.0.16.orig/configure.ac libcm-0.0.16/configure.ac
--- libcm-0.0.16.orig/configure.ac	2005-12-06 00:51:41.000000000 +0100
+++ libcm-0.0.16/configure.ac	2006-02-19 14:23:32.243530832 +0100
@@ -35,6 +35,30 @@
 # Check for dependencies
 PKG_CHECK_MODULES(DEP, glib-2.0 x11 xcomposite xdamage xtst)
 
+# AC_CHECK_LIB(libGL.so, glXCreateContext
+
+AC_CHECK_LIB(GL, glXCreateContext, HAVE_LIBGL=yes, HAVE_LIBGL=no)
+
+if test "x$HAVE_LIBGL" = "xno"; then
+   AC_MSG_ERROR(libGL not found);
+else
+   DEP_LIBS="$DEP_LIBS -lGL"
+fi
+
+AC_CHECK_LIB(GLU, gluOrtho2D, HAVE_LIBGLU=yes, HAVE_LIBGLU=no)
+
+if test "x$HAVE_LIBGLU" = "xno"; then
+   AC_MSG_ERROR(libGLU not found);
+else
+   DEP_LIBS="$DEP_LIBS -lGLU"
+fi
+
+AC_CHECK_LIB(GL, glXBindTexImageEXT, [AC_DEFINE([HAVE_TFD], [], [Define if we have TFD extension])])
+
+if test "x$HAVE_TBD" = "xyes"; then
+   AC_MSG_ERROR([Texture from drawable extension is needed])
+fi
+
 AC_SUBST(DEP_CFLAGS)
 AC_SUBST(DEP_LIBS)
 		  
